<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetVotesHistory</class>
 <widget class="QWidget" name="WidgetVotesHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#WidgetVotesHistory {
  border: 1px solid rgb(151,167, 187);
}
QWidget {
  color: white;
  background-color: rgb(28,52,62);
}
QLineEdit {
  background-color: rgb(101,117, 137);  
}
QPushButton {
  background-color: rgb(111,127, 147);
  border-style: outset;
  min-width: 3em;
  padding: 6px;
}
QPushButton:hover {
  background-color: rgb(151,167, 187);
}
QPushButton[enabled=&quot;false&quot;] {
  background-color: rgb(31,47,67);
}
QGroupBox {
  border-radius: 0px;
  border: 0px solid rgb(131, 147, 167);
  background-color: transparent; /*rgb(80,112,125);*/
}
QListWidget {
  background-color: transparent; 
}
QTextEdit {
  background-color: rgb(101,117, 137);  
}
QPlainTextEdit {
  background-color: rgb(101,117, 137);  
}
QDateTimeEdit {
  background-color: rgb(101,117, 137);  
  border-radius: 0;
  padding: 6px;
}
QPushButton {
  background: rgb(101,117, 137);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0,0">
          <item>
           <widget class="QGroupBox" name="groupBoxTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>70</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>12</number>
             </property>
             <item>
              <widget class="QLabel" name="labelFrom">
               <property name="text">
                <string>From</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateTimeEdit" name="dateTimeEditBegin">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="displayFormat">
                <string>dd.MM.yyyy</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>10</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="labelTo">
               <property name="text">
                <string>To</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateTimeEdit" name="dateTimeEditEnd">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="displayFormat">
                <string>dd.MM.yyyy</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>10</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonUpdate">
               <property name="text">
                <string>-&gt;</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelVotes">
            <property name="text">
             <string>Votes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelLine">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>1</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>1</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
  background-color: rgb(131,147, 167);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBoxVotes">
            <property name="styleSheet">
             <string notr="true">QGroupBox {
  border: 0px solid rgb(131,147, 167);
}</string>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QListWidget" name="listWidgetVotes">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="showDropIndicator" stdset="0">
                <bool>false</bool>
               </property>
               <property name="verticalScrollMode">
                <enum>QAbstractItemView::ScrollPerPixel</enum>
               </property>
               <property name="horizontalScrollMode">
                <enum>QAbstractItemView::ScrollPerPixel</enum>
               </property>
               <property name="resizeMode">
                <enum>QListView::Adjust</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
